core/src/main/java/com/amplifyframework/logging/JavaLoggingPlugin.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.logging;
+
+import android.content.Context;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONObject;
+
+/**
+ * Implementation of the {@link LoggingCategoryBehavior} designed for use by unit tests,
+ * which emits logs via {@code System.out.println()}.
+ */
+public final class JavaLoggingPlugin extends LoggingPlugin<Void> {
+    private static final String AMPLIFY_NAMESPACE = "amplify";
+    private final LogLevel defaultLoggerThreshold;
+
+    /**
+     * Creates a logging plugin using {@link LogLevel#INFO} as the default
```

Comment:
The comment says `LogLevel.INFO` is used as the default logging threshold, but the constructor itself uses `LogLevel.VERBOSE`.